plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/world/PlayEffectCommand.java

```diff
@@ -238,7 +238,7 @@ public void execute(ScriptEntry scriptEntry) throws CommandExecutionException {
 
         for (dLocation location : locations) {
             // Slightly increase the location's Y so effects don't seem to come out of the ground
-            location.add(0, 1, 0);
+            location.clone().add(0, 1, 0);

```

Comment:
No. That's not... no. You turned code that did a thing into code that does nothing lol.